### DISCUSSION TITLE
Regarding license

### DISCUSSION THREAD

**Discussion by acrlakshman** [sidbannet/converge_cfd_user#9] [General]
Hi @sidbannet, this is a minor thing from my side, do you want to stick with GPL or do you have any plans to change license to those that are less viral, like MIT, BSD, before this project gets a release tag? I am fine either way, just wanted to have a little conversation with regard to licensing.

**Comment by sidbannet** [comment]
@acrlakshman thanks. I have GPL for simplicity. Particularly I like it's [feature](https://choosealicense.com/) of sharing but restricting sharing or distribution closed source version. Let me know if this restricts in anyway to contribute here.

**Reply by acrlakshman** [reply]
No, this does not restrict me. If needed, in future I can find some other way to fit the needs of this project, for instance, I might release another python package with less viral license and embed that in this work.
